neo4j-5.5.0-default.conf

```diff
@@ -0,0 +1,13 @@
+
+db.tx_log.rotation.retention_policy=100M size
+
+server.memory.pagecache.size=512M
+
+server.default_listen_address=0.0.0.0
+
+server.discovery.advertised_address=7945ab27216c:5000
+
+server.cluster.advertised_address=7945ab27216c:6000
+
+server.cluster.raft.advertised_address=7945ab27216c:7000
+server.directories.logs=/logs
```

Comment:
@christabone what happens with this file? Can it be left out?

Reply:
It's just for reference purposes. I can remove it if you'd like? Let me know.

Reply:
Its fine... its just not clear when looking at this file on how its used, or if it needs to be used.

Reply:
I added a line in the readme for clarity.